R/pool.R

```diff
@@ -186,7 +259,11 @@ Pool <- R6Class("Pool",
           NULL
         )
         if (!is.null(removeFrom)) {
-          rm(list = id, envir = removeFrom)
```

Comment:
Worth checking that `removeFrom` matches with `pool_metadata$state`? They should always match now, right?